save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -28,8 +30,9 @@ class WarnPlugin(testConfig: TestConfig) : Plugin(testConfig) {
 
     override fun execute(): Sequence<TestResult> {
         val warnPluginConfig = testConfig.pluginConfigs.filterIsInstance<WarnPluginConfig>().single()
+        val generalConfig = testConfig.pluginConfigs.filterIsInstance<GeneralConfig>().single()
```

Comment:
Let's make general config optional here, and if it is null, then the default value for `ignore...` parameter is used